frontend/public/components/storage/create-pvc.tsx

```diff
@@ -36,6 +36,9 @@ const provisionerAccessModeMapping = {
   'kubernetes.io/storageos' : ['ReadWriteOnce'],
 };
 
+// These provisioners not support ROX OCS 4.2
+const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
```

Comment:
It would be good to keep this list in one place instead of defining it in two separate places. I'd recommend adding a generic helper to check for these provisioners and using it wherever we need it (here and for add capacity).

Reply:
Can we export a generic function from here and import it in capacity and dashboards.
Or I dont know if there is some better place to put in.
```
export const isCephProvisioner = (provisioner: string) => {
  const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
  return cephStorageProvisioners.some((p: string) => provisioner.includes(p));
};
```

Reply:
It probably should go in `console-shared`. There are other utils there you can look at.

The other place that would be reasonable is storage-class.tsx where `isDefaultClass` is defined. But console-shared is probably better.

Reply:
@spadgett @afreen23 there are two PRs for 4.3 for refactoring storage dropdown where the same filter is used #2826 and #2835, since this bug needs to go in with 4.2.z, can we consider taking a refactor PR for this after the above two PR merges ?

Reply:
@spadgett @afreen23 updated this.